dags/environment/__init__.py

```diff
@@ -0,0 +1,29 @@
+import importlib
+
+from common import OTAP_ENVIRONMENT
```

Comment:
Is this from DaVe repo? I was looking for it [here](https://dev.azure.com/CloudCompetenceCenter/Dataverwerking/_git/airflow-operators?path=/src/common) but could not find it. Also, where is `OTAP_ENVIRONMENT` set?

Reply:
An example where they use it is in the code that sends Slack messages: https://github.com/Amsterdam/dataservices-airflow/blob/master/src/plugins/slack_message_operator.py#L54